ci/test/06_script_b.sh

```diff
@@ -21,8 +21,6 @@ if [ -n "$USE_VALGRIND" ]; then
   END_FOLD
 fi
 
-bash -c "${CI_WAIT}" &  # Print dots in case the tests take a long time to run
-
```

Comment:
This deletion is also necessary?

Reply:
`CI_WAIT` no longer exists and this should be a no-op

Reply:
Yes, `CI_WAIT` was dropped from the `.travis.yml` in #18929.